src/pages/PaymentsPage.js

```diff
@@ -1,21 +1,31 @@
 import React, { Component } from "react";
+import { bindActionCreators } from "redux";
 import { connect } from "react-redux";
-import { injectIntl } from 'react-intl';
+import { injectIntl } from "react-intl";
 import { withTheme, withStyles } from "@material-ui/core/styles";
-import { withModulesManager, withHistory } from "@openimis/fe-core"
+import {
+  withModulesManager,
+  withHistory,
+  clearCurrentPaginationPage,
+} from "@openimis/fe-core";
 import PaymentSearcher from "../components/PaymentSearcher";
 
 const styles = theme => ({
     page: theme.page,
     fab: theme.fab
 });
 
-
 class PaymentsPage extends Component {
     // onAdd = () => {
     //     historyPush(this.props.modulesManager, this.props.history, "payment.paymentNew");
     // }
 
+    componentDidMount = () => {
+        const moduleName = "payment";
```

Comment:
I wonder if such kind of variable named in such convention: 
`const MODULE_NAME = "payment";`
And also I wonder if should we add such kind of const into separate file or at the beginiing of this component. What do you think about this? @olewandowski1 . For me this is important const that indicates  the module in your solution.